static/js/tools/timeline/page.tsx

```diff
@@ -15,250 +15,116 @@
-      Promise.all([placesPromise, validStatsVarPromise]).then((values) => {
-        this.setState({
-          placeIdNames: values[0],
-          statsVarValid: values[1],
-        });
-      });
-      this.params.setUrlPlaces();
-    }
-  }
-
-  // remove one place
-  private removePlace(place: string): void {
-    if (this.params.removePLace(place)) {
-      const tempPlace = this.state.placeIdNames;
-      delete tempPlace[place];
-      getStatsVar(this.params.placeDcids).then((data) => {
-        this.setState({
-          placeIdNames: tempPlace,
-          statsVarValid: data,
-        });
-      });
-      this.params.setUrlPlaces();
-    }
-  }
-
-  // call back function passed down to menu for getting statsVar titles
-  setStatsVarTitle(statsVarId2Title: Record<string, string>): void {
-    for (const id in statsVarId2Title) {
-      // remove title of unselected statsVars
-      if (!Object.keys(this.state.statsVarNodes).includes(id)) {
-        delete statsVarId2Title[id];
-      }
-    }
-    for (const id in this.state.statsVarNodes) {
-      if (!(id in statsVarId2Title)) {
-        statsVarId2Title[id] = id;
-      }
-    }
-    this.setState({
-      statsVarTitle: statsVarId2Title,
-    });
-  }
-
-  // set PerCapita for a chart
-  setChartPerCapita(mprop: string, pc: boolean): void {
-    if (this.params.setChartPC(mprop, pc)) {
-      this.setState({
-        chartOptions: _.cloneDeep(this.params.chartOptions),
-      });
-      this.params.setUrlChartOptions();
-    }
+    );
   }
 
   render(): JSX.Element {
-    let statsVarFilter: StatsVarFilterInterface;
-    if (Object.keys(this.state.placeIdNames).length === 0) {
-      statsVarFilter = new NoopStatsVarFilter();
-    } else {
-      statsVarFilter = new TimelineStatsVarFilter(this.state.statsVarValid);
+    const numPlaces = Object.keys(this.placeName).length;
+    const numStatsVarInfo = Object.keys(this.statVarInfo).length;
+    const namedPlaces: NamedPlace[] = [];
+    for (const place in this.placeName) {
+      namedPlaces.push({ dcid: place, name: this.placeName[place] });
     }
-    const numPlaces = Object.keys(this.state.placeIdNames).length;
-    const numStatsVarInfo = Object.keys(this.state.statsVarInfo).length;
+    const statVars = Array.from(getTokensFromUrl("statsVar", statVarSep));
     return (
       <div>
-        <div className="explore-menu-container" id="explore">
+        <div className="stat-var-hierarchy-container" id="explore">
           <div id="drill-scroll-container">
             <div className="title">Select variables:</div>
-            <Menu
-              selectedNodes={this.state.statsVarNodes}
-              statsVarFilter={statsVarFilter}
-              setStatsVarTitle={this.setStatsVarTitle.bind(this)}
-              addStatsVar={this.addStatsVar.bind(this)}
-              removeStatsVar={this.removeStatsVar.bind(this)}
-            ></Menu>
+            <div style={{ width: "350px" }}>
```

Comment:
is this something we can set in css?

Reply:
This is not needed at all.. Removed and updated css